packages/backend/src/routes/checkout.js

```diff
@@ -1,80 +1,89 @@
 import express from 'express';
-import { db } from '../lib/database';
+import status from 'http-status';
+import { db, dbQueryPromise } from '../lib/database';
 import { getDistance } from '../lib/distanceMatrix';
+import { ticketTransaction } from '../lib/bank';
+import {
+  checkCreditCard,
+  validCreditCard,
+  getCardNumber,
+} from '../lib/creditcard';
+import { getSellerId, getPrice } from '../lib/tickets';
 
 const router = express.Router();
 
-router.post('/buy/submit', (req, res, next) => {
+router.post('/shipping/submit', async (req, res, next) => {
   if (req.session && req.session.userId) {
     const ticketInfo = {
-      boughtUserId: req.session.id, // check
       deliveryMethod: req.body.deliveryMethod,
       // address: req.body.address,
       ticketId: req.body.ticketId,
     };
 
-    db.query(
+    dbQueryPromise(
```

Comment:
This route is not executing the transaction, therefore we shouldn't update the database. I think you meant something like calculating price, in which case you should use the sale-charge route.